lib/MySQLProtocolParser.pm

```diff
@@ -75,7 +75,7 @@ use constant {
    COM_BINLOG_DUMP         => '12',
    COM_TABLE_DUMP          => '13',
    COM_CONNECT_OUT         => '14',
-   COM_REGISTER_SLAVE      => '15',
+   COM_REGISTER_REPLICA    => '15',
```

Comment:
The packet exists on 8.0 and 8.4, but not 5.7
I don't find any code usage of it in any lib/bin though